src/webprojects/fxa.md

```diff
@@ -15,98 +13,99 @@ The above could change over time as the team is planning to migrate .po files to
+* Login screen.
+* Email notification.
+* Check all the setting configuration buttons and the page or expanded window each page leads to.
+* Translation is of good quality and makes sense in context.
 * No broken UI, especially in a mobile environment. Text wraps or line breaks correctly.
-* Terms are consistent between pages
-* Basic font support
-* Error and warning messages
+* Terms are consistent between pages.
+* Basic font support.
+* Error and warning messages.
 
-#### Functionality testing:
+#### Functionality testing
 
-* Configure the setting. Go through all of the categories
-* Check for email notifications with each of the initial setups
-* Make changes to some of the setting, check for possible error message or warning; check for followup confirmation emails
+* Configure the Settings. Go through all of the categories.
+* Check for email notifications with each of the initial setups.
+* Make changes to some of the settings, check for possible error messages or warnings; check for followup confirmation emails.
 
 ### Testing the subscription platform
 
-The subscription platform (or sub plat) consists of a few features. Right now only the `Payment and subscription creation` is available for testing:
+The subscription platform (or sub plat) consists of a few features. Right now only the _Payment and subscription creation_ is available for testing:
 
-* `Payment and subscription creation`: The initial payment funnel where users enter their credit card information to subscribe to a product. Mozilla partners with Stripe to handle this
-* `Billing`: Ongoing billing and receipt emails once a subscription is entered
-* `Subscription management UI`: Settings that allow the user to update their payment information
+* _Payment and subscription creation_: The initial payment funnel where users enter their credit card information to subscribe to a product. Mozilla partners with Stripe to handle this.
+* _Billing_: Ongoing billing and receipt emails once a subscription is entered.
+* _Subscription management UI_: Settings that allow the user to update their payment information.
 
 Payment testing is tied to the language of user preference and location. Payment service is market specific. For instance, the product is available in German, but the service is available in Germany but not in Switzerland.
 
-#### Before testing:
+#### Before testing
 
 * Have a Firefox account. If you don’t have one, the staging link will take you to the signup process.
 * Choose a Firefox browser for: desktop (Windows only), Android, or iOS. Support for macOS and Linux is coming soon.
-* Set the language you are testing in your browser.
+* If you are not already using the browser in the language you are going to test in, type in the URL field `about:Preferences`, you will be taken to the preferences setting page. Scroll down to the **Language and Appearance** section, under **Language**, click on the **Choose** button. Select the language you want to test in.
```

Comment:
`about:preferences`, lowercase.

But you should point to SUMO more than giving step by step instructions, since they have instructions and screenshots, and they're up to date
https://support.mozilla.org/kb/choose-display-languages-multilingual-web-pages